### ISSUE TITLE
Android 11 Force Closed

### ISSUE THREAD

**Issue by sankarwap** [PatilShreyas/EasyUpiPayment-Android#97]
AppNotFoundException: No UPI app exists on this device to perform this transaction

In App module manifest add the  below queries working fine.
`<queries>`
        `<package android:name="in.amazon.mShop.android.shopping" />`
        `<package android:name="com.google.android.apps.nbu.paisa.user" />`
        `<package android:name="in.org.npci.upiapp" />`
        `<package android:name="com.phonepe.app" />`
       ` <package android:name="net.one97.paytm" />`
`</queries>`

**Comment by ShankarKakumani** [comment]
Yeah, From android 11, we need to add package names of the applications in the manifest

**Comment by PatilShreyas** [comment]
Thanks for reporting this issue. Looks like this is happening for devices having Android 11. Currently, I don't have any device having Android 11 to test this change. I've tried to fix this change and attached APK here 👇🏻.

[***Download Sample APK***](https://github.com/PatilShreyas/EasyUpiPayment-Android/raw/v3.0.3-dev/app/app-debug.apk)

It's my humble request to test the attached APK and please let me know if it works as expected. If it works, I'll be happy to release the new version `v3.0.3`.

**Comment by PatilShreyas** [comment]
As @sankarwap mentioned

I think a temporary solution would be declaring package visibility needs from the application side where you're integrating this library. For e.g. you can add package names for UPI apps

```xml
<queries>
    <package android:name="in.amazon.mShop.android.shopping" />
    <package android:name="in.org.npci.upiapp" />
    <package android:name="com.google.android.apps.nbu.paisa.user" />
    <package android:name="com.phonepe.app" />
    <package android:name="net.one97.paytm" />
</queries>
````

For now, you can allow only specific apps for payment instead of choosing among any of them. I'll try to fix this issue ASAP from the library as well.

**Comment by PatilShreyas** [comment]
Thanks, this has been fixed in the latest release `v3.0.3`.
